CharacterCreation/01_Races.txt

```diff
@@ -226,7 +227,7 @@ Seadevil
 -1 to base CHA, -1 to base FOR.

```

Comment:
I second Turtlelord here, can you chance "FOR" to "FORT" for consistencies sake?

Reply:
I think I'll change, not chance ;) Will do!

Reply:
toupee